students/WesleyM/Lesson3/mailroom.py

```diff
@@ -0,0 +1,47 @@
+donors = [
+  ["Alice", 20],
+  ["Bob", 100],
+  ["Charles", 30, 50, 10],
+  ["Denise", 10, 5],
+  ["Edward", 25, 20, 20, 20, 10],
+  ]
+
+def usein():
```

Comment:
Python style guide uses snake case for functions and variable names and it's a good idea to avoid abbreviations/short hand for readability, so I'd probably call this function something like `user_input`